src/ansys/fluent/core/launcher/slurm_launcher.py

```diff
@@ -2,12 +2,22 @@
 
 Examples
 --------
->>> slurm = pyfluent.launch_fluent(scheduler_options={"scheduler": "slurm"})
+>>> # The keys scheduler_headnode, scheduler_queue and scheduler_account are optional
+>>> # and should be specified in a similar manner to Fluent's scheduler options.
```

Comment:
I believe that removing this from the code block, and changing it from a code comment to text, should work better:
```suggestion
Note that the keys ``scheduler_headnode``, ``scheduler_queue`` and ``scheduler_account`` are 
optional and should be specified in a similar manner to Fluent's scheduler options.

```

Reply:
Should we instead link to the more complete documentation at `doc/source/user_guide/launching_ansys_fluent.rst`, rather than repeating this info here?

Reply:
I feel as the note explains the arguments of the following code, it is better to place it close to the code. I've moved it from comment to text.